support/mock/mockrt.go

```diff
@@ -660,6 +667,30 @@ func (rt *Runtime) BatchVerifySeals(vis map[addr.Address][]proof.SealVerifyInfo)
 	return nil, nil
 }
 
+func (rt *Runtime) ExpectAggregateVerifySeals(agg proof.AggregateSealVerifyProofAndInfos, err error) {
+	rt.expectAggregateVerifySeals = &expectAggregateVerifySeals{
+		agg.Infos, agg.Proof, err,
+	}
+}
```

Comment:
Nit: this file is organised with all the expectation methods grouped together, and the interface methods grouped separately. Follow that grouping for now, and re-arrange everything in a separate PR if you wish.